.github/workflows/ci.yml

```diff
@@ -109,6 +110,8 @@ jobs:
       run: |
         set -ex
         make regenerate-example-migrations
+        # TODO: This file is created by one of the tests on Windows
+        rm -f butane_core/sqlite
```

Comment:
I'll solve this on a different branch afterwards - need to figure out which one(s) it is, and delete the file as part of those tests.